splunklib/client.py

```diff
@@ -2887,6 +2887,38 @@ def unpause(self):
         self.post("control", action="unpause")
         return self
 
+    def await_completion(self, sleep_interval, callback=lambda done, progress: False):
```

Comment:
@cheddar how do you plan to call this function? We haven't use callbacks as a pattern in the SDK, and don't recommend we do so here.
Also the word `await` implies some async behavior which is not preset, so I would suggest renaming this function to `wait_for_completion()`